script.js

```diff
@@ -32,62 +32,86 @@ pageBody.appendChild(mainPage);
 const bookForm = document.getElementById('add-book');
 
-let books = [];
+class BookArray {
+  constructor() {
+    this.books = [];
+  }
+
+  createBook(title, author) {
+    const book = {
+      title,
+      author,
+    };
+    this.books.push(book);
+  }
+
+  removeBookFromArray(bookNum) {
+    this.books.splice(bookNum, 1);
+  }
+}
 
-function displayBooks() {
-  books.forEach((book, bookNum) => {
-    const bookDiv = document.createElement('div');
-    bookDiv.classList.add(`${bookNum}`);
+const booksObj = new BookArray();
+// eslint-disable-next-line prefer-destructuring
+let books = booksObj.books;
 
-    bookDiv.innerHTML = `
-      <h2></h2>
-      <h3></h3>
-      <button type="button" onmousedown="removeBook(${bookNum})">Remove</button>`;
+function addBookToPage(title, author, bookNum) {
+  const bookDiv = document.createElement('div');
+  const bookId = bookNum;
+  bookDiv.classList.add(`book-${bookId}`);
 
-    bookDiv.children[0].textContent = `${book.title}`;
-    bookDiv.children[1].textContent = `${book.author}`;
+  bookDiv.innerHTML = `
+    <h2></h2>
+    <h3></h3>
+    <button type="button" onmousedown="removeBook(${bookId})">Remove</button>`;
 
-    bookList.appendChild(bookDiv);
-  });
-}
+  bookDiv.children[0].textContent = `${title}`;
+  bookDiv.children[1].textContent = `${author}`;
 
-function refreshBookList() {
-  bookList.innerHTML = '';
-  displayBooks();
+  bookList.appendChild(bookDiv);
 }
 
-function addBook() {
-  const currentTitle = bookForm.elements.title.value;
-  const currentAuthor = bookForm.elements.author.value;
+function displayStoredBooks() {
+  books.forEach((book, bookNum) => {
+    booksObj.createBook(book.title, book.author);
+    addBookToPage(book.title, book.author, bookNum);
+  });
+}
 
-  books.push(
-    {
-      title: `${currentTitle}`,
-      author: `${currentAuthor}`,
-    },
-  );
-  refreshBookList();
+function displayCurrentBooks() {
+  books.forEach((book, bookNum) => {
+    addBookToPage(book.title, book.author, bookNum);
+  });
 }
 
 function saveBooksToLocalStorage() {
+  books = booksObj.books;
   localStorage.setItem('books', JSON.stringify(books));
 }
 
 // eslint-disable-next-line no-unused-vars
-function removeBook(bookNumber) {
-  books.splice(bookNumber, 1);
-  refreshBookList();
+function removeBook(bookId) {
+  booksObj.removeBookFromArray(bookId);
   saveBooksToLocalStorage();
+
+  bookList.innerHTML = '';
+  displayCurrentBooks();
 }
 
 bookForm.addEventListener('submit', (event) => {
   event.preventDefault();
-  addBook();
+
+  const currentTitle = bookForm.elements.title.value;
+  const currentAuthor = bookForm.elements.author.value;
+
   bookForm.reset();
+
+  booksObj.createBook(currentTitle, currentAuthor);
   saveBooksToLocalStorage();
+  addBookToPage(currentTitle, currentAuthor, books.length);
+  window.location.reload();
 });
 
 if (localStorage.getItem('books')) {
   books = JSON.parse(localStorage.getItem('books'));
```

Comment:
why not set the objects array here instead of adding it manually in a loop in the display stored books function?